backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/blobs/CassandraBlobMapper.java

```diff
@@ -57,16 +60,24 @@ public CassandraBlobMapper(Session session) {
     public CompletableFuture<UUID> write(byte[] data) {
         UUID uuid = UUIDs.timeBased();
         return FluentFutureStream.of(
```

Comment:
I know it's a POC but it's not meant to be a POC for ever. By the way the quality is awesome for a POC.